### ISSUE TITLE
Can I make a content pack for FlansMod?

### ISSUE THREAD

**Issue by slyfoxD** [FlansMods/FlansMod#1194]
I would like to offer my help to the development of Flans Mod.
I can't do anything major yet, but I've made some recent strides in Minecraft modding and I would like to use Flans Mod as an outlet for my enthusiasm. 

Is there any hope of new development?
Is there any way I can be helped with my request?  
If I accomplish something worth releasing can I ?

Alternatively, I would just like to know about the state of Flans Mod now.

**Context**
I grew up playing with Flans Mod. 
Current events have me stuck at home without a creative outlet, this led me to learning Minecraft modding.
I have little coding experience, but I'm investing time in changing that.

**Comment by Liruxo** [comment]
First of all, the following only represents my opinions and my point of view, however i try to represent the other involved parties to the best of my ability.

Since you gave a bit of interfering intentions, ill first address making content packs:
In general you are completely free to create content packs and distribute them however you please. However, creating content packs is whole world for it self, not having much in common with minecraft mod development. This github mainly focuses on maintaining the current official content packs, and developing and maintaining the FlansMod java source code. If you want to create completely  new and exciting content packs i can refer you to a discord server where you can get help from much more experienced model-makers and content-creators.

Since you mentioned you want to "code", i assume you mean you are interested in Forge-Java development too. In this case this is the right place. We, or at least i, are happy to help you with problems you encounter while developing FlansMod, but keep in mind that we can and will not provide basic Java and/or Forge-Modding tutorial like support, there are other places for such things.

On the state of FlansMod from my view: This projects currently suffers from lacking contributions, for example in my case, i do currently not posses the time for the development. However, i did not abandon this project, at this time i still plan further development, such as a port onto newer minecraft version. However, as mentioned, this is a future project of mine without any guarantees.

**Comment by slyfoxD** [comment]
Thanks for replying. 
I apologize for not clarifying my intentions.
I eventually want to develop my own mods evolving guns like FlansMod or even just as a content pack for it. 
However it's my understanding that this is impossible unless I know Java and Forge.
So that's what I've been learning, and I've been using any resources I can find online to get me to the point  of being able to make my own additions to the game. Currently I'm using Eclipse Java to do the most basic code to add items, blocks, and recipes into the game. 
Eventually I realized that FlansMod already had the resources I needed to make and add guns into the game. After some practice I got the C96 model already existing in the code to appear in the game as a usable gun. It wasn't too hard it just took time to create and test if it worked right, about 2 hours of figuring out what the mod needed to have in order to recognize a new gun. Best of all the C96 model existing in the code was gorgeous, way better than the model I created for my tests.

No doubt someone with more experience in FlansMod could get any model to work just fine, my point being I have lots of spare time to spend on creative interests such as getting a gun model to work. 

So in short:
I would love to be in contact with anyone who can help me or give me directions as I figure out what I need to accomplish my modding goals.
I may be wrong but I believe working with FlansMod will get me closer to my goal.
Thanks again for replying to my questions. I hope to be of more use in the future especially when it comes to the development of Flans Mod.
As of right now I can only report on some small bugs I found while doing some playing with the Mod.

**Comment by Liruxo** [comment]
If you are still interested, you can join the SMP Discord Server here https://discord.gg/FYYyysq
Additionally you then can send me a DM on discord (same name), so i can invite you to the "Flan's Games"-Discord
